ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -94,6 +94,7 @@ object LedgerApiTestTool {
 
     val testsToRun = Tests.all.filterKeys(included -- config.excluded)
 
+
```

Comment:
I feel like you probably meant to do more here. As it is you don't seem to use those new fields at all.

Reply:
Oh, sorry, didn't see this was a draft. Carry on!